Assets/Scripts/Game/Questing/QuestMCP.cs

```diff
@@ -258,6 +257,22 @@ public override string Direction()
 
                 return TextManager.Instance.GetLocalizedText("resolvingError");
             }
+
+            private static FactionFile.FactionIDs GetRandomDivine()
+            {
+                switch (UnityEngine.Random.Range(0, 9))
+                {
+                    case 0: return FactionFile.FactionIDs.Arkay;
+                    case 1: return FactionFile.FactionIDs.Zen;
+                    case 2: return FactionFile.FactionIDs.Mara;
+                    case 3: return FactionFile.FactionIDs.Ebonarm;
```

Comment:
Not sure about Ebonarm being in the random selection.
We could go full "Daggerfall urban legend" and make Ebonarm the "FG with Temple building type" deity, instead of a random selection. Having a random deity there might get them confused with religious knight orders

Reply:
I'm not familiar enough with the lore to be able to say which divines would be inappropriate to mention in a Fighters Guild hall.

I went with the original code comment "Better just to provide something than let let game loop crash".